src/modules/toolbox/components/shareToolContent/ShareToolDialog.js

```diff
@@ -51,10 +54,20 @@ export class ShareToolDialog extends BaElement {
     `;
 	}
 
-	async _copyValueToClipboard(value) {
-		await this._shareService.copyToClipboard(value).then(() => { }, () => {
```

Comment:
Why did we have to change this?

Reply:
because:
1) the call to the shareService and to the notifications are not yet implemented
2) we dont want to use Promise API like "then" anymore :-)